guides/micronaut-oauth2-linkedin/java/src/main/java/example/micronaut/LinkedInMe.java

```diff
@@ -10,16 +11,17 @@
 
     @NonNull
     @NotBlank
-    private String id;
+    private final String id;
 
     @NonNull
     @NotBlank
-    private String localizedFirstName;
+    private final String localizedFirstName;
 
     @NonNull
     @NotBlank
-    private String localizedLastName;
+    private final String localizedLastName;
 
+    @Creator
```

Comment:
Is `@Creator` necessary? If you want to keep, we need to add a callout and explain it to the user.